apps/namadillo/src/hooks/useTransaction.tsx

```diff
@@ -120,80 +113,78 @@ export const useTransaction = <T,>({
     mutationFn: async (
       additionalParams: Partial<BuildTxAtomParams<T>> = {}
     ) => {
-      invariant(
```

Comment:
Nothing really changed here, I just wrapped with a try catch. The relevant part is related to error throwing below.